pkg/sql/show_tenant.go

```diff
@@ -70,13 +73,13 @@ func (p *planner) ShowTenant(ctx context.Context, n *tree.ShowTenant) (planNode,
 	if err != nil {
 		return nil, err
 	}
-
 	node := &showTenantNode{
 		tenantSpec:           tspec,
 		withReplication:      n.WithReplication,
 		withPriorReplication: n.WithPriorReplication,
 		withCapabilities:     n.WithCapabilities,
 		initTenantValues:     true,
+		statementTime:        p.EvalContext().GetStmtTimestamp(),
```

Comment:
hm, I think this is being evaluated at prep, rather than exec, now? is there not an eval ctx you can use during exec?

Reply:
I don't know for sure, but I think the runParams to startExec are what you're supposed to use to evaluate expressions during exec

Reply:
moved the call to exec. unclear if it makes a difference.